modules/tinymce/src/core/main/ts/api/dom/Serializer.ts

```diff
@@ -105,7 +105,45 @@ const DomSerializer = (settings: DomSerializerSettings, editor?: Editor): DomSer
 
     getNodeFilters: domSerializer.getNodeFilters,
 
-    getAttributeFilters: domSerializer.getAttributeFilters
+    getAttributeFilters: domSerializer.getAttributeFilters,
+
+    /**
+     * Removes a node filter function or removes all filter functions from the parser used by the serializer for the node names provided.
+     *
+     * @method removeNodeFilter
+     * @param {String} name Comma separated list of node names to remove filters for.
+     * @param {Function} callback Optional callback function to only remove a specific callback.
+     * @example
+     * // Remove a single filter
+     * serializer.removeNodeFilter('p,h1', (nodes, name) => {
+     *   for (var i = 0; i < nodes.length; i++) {
+     *     console.log(nodes[i].name);
+     *   }
+     * });
+     *
+     * // Remove all filters
+     * serializer.removeNodeFilter('p,h1');
+     */
+    removeNodeFilter: domSerializer.removeNodeFilter,
+
+    /**
+     * Removes an attribute filter function or removes all filter functions from the parser used by the serializer for the attribute names provided.
+     *
+     * @method removeAttributeFilter
+     * @param {String} name Comma separated list of attribute names to remove filters for.
+     * @param {Function} callback Optional callback function to only remove a specific callback.
+     * @example
```

Comment:
Same here remove or simplify the example since passing in a function like that needs to be a reference to an existing function.